strapi/strapi
# chore(releases): setup releases plugin

### What does it do?

Sets up the basic shell for the releases plugin

### What it does NOT do

- Handle enterprise edition

### Why is it needed?

To start working in parallel on releases

### How to test it?

- yarn setup
- build and run the examples/getstarted
- you should have a server log for "Releases server loaded..." and an admin log for "Releases admin loaded..."
